src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -776,4 +783,25 @@ public static int getPoLineLimit(JsonObject config) {
   public static CompositePurchaseOrder convertToCompositePurchaseOrder(JsonObject poJson) {
     return poJson.mapTo(CompositePurchaseOrder.class);
   }
+
+  public static boolean inventoryUpdateNotRequired(CompositePoLine compPOL) {
```

Comment:
This conditional is quite large... We might consider breaking it up, or at least adding additional comments

Reply:
conditions updated for better readability